SenderEmail/src/main/java/com/example/SenderEmail/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,29 @@
+package com.example.SenderEmail.exception;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.server.ResponseStatusException;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
+
+ private static final Logger logger = LoggerFactory.getLogger(GlobalExceptionHandler.class) ;
```

Comment:
- Change `logger` to be `LOGGER`.
- Reason: naming convention for `final` fields should be in capital.